### ISSUE TITLE
Strange pronunciation of multiple punctuation marks  

### ISSUE THREAD

**Issue by SergTSA** [RHVoice/RHVoice#284]
In Samsung TalkBack function multiple punctuation marks are pronounced like as "Символов Восклицательный  знак три". 

Android 11, Samsung Note 20 Ultra

**Comment by alex19EP** [comment]
can you provide specific steps to repeat the error?

**Comment by SergTSA** [comment]
First you activate Talkback in Samsung Android11 by pressing Vol+ and Main. Then in browser open any text containing multiple punctuation marks with text - [for example](https://author.today/reader/113107/897645) 
you can see string 
`засранка, новую куртку!!!`
the last three explanation marks are read as 
`"Символов Восклицательный знак три`

**Comment by alex19EP** [comment]
Do I understand correctly that this problem does not happen on android10 and not on samsung phones?

is punctuation reading enabled in talkback settings?

**Comment by SergTSA** [comment]
It's not the same at Android 9 redmi note 5 phone but RHVoice version was 1.2.2, the next did not updated. When 1.2.3 was installed  it still did not anounced triple marks. The Android 10 in Samsung had another function smth like display-reader, not talkback. And now I can't check. 

Note 10+ and Note 20U from samsung both anounced multiple marks as in Chrome as in Mi brousers.

**Comment by zstanecic** [comment]
@alex19EP 
I cannot reproduce this issue, even with talkback, and or with jieshuo screenreader. Note that i've checked this with latest version for Android, on android 11, RHVoice 1.2.3/1.2/4.

**Comment by alex19EP** [comment]
thank you for info @zstanecic.

can @SergTSA  add something?

**Comment by zstanecic** [comment]
Hi! Is there any progress on this issue? If not, is it secure to be closed, because of the lost of contact with the author?
